e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -215,6 +239,11 @@ CleanUp-Certs
 
 Write-Host "`nGenerating self signed certs."
 
+# Generate the certificates used by both IoT Hub and DPS tests.
```

Comment:
This block has only been arranged and additional comments have been added. No command has been removed in this PR.